### ISSUE TITLE
I can not set batch_size more than 4,my GPU is GTX980 Ti, I do not why, so when i train the model, the loss  just decrease to 10.

### ISSUE THREAD

**Issue by Suodislie** [georgesung/ssd_tensorflow_traffic_sign_detection#7]

**Comment by georgesung** [comment]
The GTX 980 Ti has 6GB VRAM which is less than the 8GB VRAM of the reference GTX 1080 I used, so that may explain why you had to use a smaller batch size. As to the loss, maybe resume learning with a lower learning rate.
